src/course/the2nd/Task2_anotherSolution.java

```diff
@@ -0,0 +1,13 @@
+package course.the2nd;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+
+public class Task2_anotherSolution {
```

Comment:
クラス名ですがキャメルケースにしましょう！
Javaの命名規則について参考情報を送ります。
https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html
```suggestion
public class Task2AnotherSolution {
```

Reply:
このようにコミットログを貼れるのでスレッドに返事してもらえればよいです！

58487045c3f160f50b7ebc4f4a7ffd0e3a3ecce1

<img width="500" alt="スクリーンショット 2022-03-27 21 45 06" src="https://user-images.githubusercontent.com/62045457/160281957-298be1b2-c250-4590-856a-293fec64a98c.png">

<img width="500" alt="スクリーンショット 2022-03-27 21 43 48" src="https://user-images.githubusercontent.com/62045457/160281891-e6273ea3-14bb-4bcd-851b-a1d5a2bad310.png">